Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -150,3 +150,31 @@ QVector<QString> Settings::GetProfiles(const InputConfig* config) const
 
   return vec;
 }
+
+bool Settings::IsLogVisible() const
+{
+  return SConfig::GetInstance().m_InterfaceLogWindow;
```

Comment:
I blanked on this before, but both of these are transient UI information and should be stored in native settings as well.